resource/ui/MainWindow.ui

```diff
@@ -130,6 +130,44 @@
             <property name="position">2</property>
           </packing>
         </child>
+        <child>
```

Comment:
Just checked and found that `Start in Tray` is a better naming for this feature. Could you please change all the occurrences?